docs/fsharp/style-guide/formatting.md

```diff
@@ -100,42 +100,52 @@ If you have a long function definition, place the parameters on new lines and in
-                                        (aThirdVeryLongParam: AVeryLongTypeThatYouNeedToUse)
-                                        =
-        // ... the body of the method follows
+    let LongFunctionWithLotsOfParameters
+        (aVeryLongParam: AVeryLongTypeThatYouNeedToUse)
+        (aSecondVeryLongParam: AVeryLongTypeThatYouNeedToUse)
+        (aThirdVeryLongParam: AVeryLongTypeThatYouNeedToUse)
+        =
+            // ... the body of the method follows
 ```
 
 This also applies to members, constructors, and parameters using tuples:
 
 ```fsharp
 type TM() =
-    member _.LongMethodWithLotsOfParameters(aVeryLongParam: AVeryLongTypeThatYouNeedToUse,
-                                            aSecondVeryLongParam: AVeryLongTypeThatYouNeedToUse,
-                                            aThirdVeryLongParam: AVeryLongTypeThatYouNeedToUse) =
-        // ... the body of the method
-
-type TC(aVeryLongCtorParam: AVeryLongTypeThatYouNeedToUse,
+    member _.LongMethodWithLotsOfParameters
+        (
+            aVeryLongParam: AVeryLongTypeThatYouNeedToUse,
+            aSecondVeryLongParam: AVeryLongTypeThatYouNeedToUse,
+            aThirdVeryLongParam: AVeryLongTypeThatYouNeedToUse
+        ) =
+            // ... the body of the method
+
+type TC
+    (
+        aVeryLongCtorParam: AVeryLongTypeThatYouNeedToUse,
         aSecondVeryLongCtorParam: AVeryLongTypeThatYouNeedToUse,
-        aThirdVeryLongCtorParam: AVeryLongTypeThatYouNeedToUse) =
-    // ... the body of the class follows
+        aThirdVeryLongCtorParam: AVeryLongTypeThatYouNeedToUse
+    ) =
+        // ... the body of the class follows
 ```
 
 If the parameters are currified or there's an explicit return type annotation, it is preferable to place the `=` character on a new line:
 
 ```fsharp
 type C() =
-    member _.LongMethodWithLotsOfParameters(aVeryLongParam: AVeryLongTypeThatYouNeedToUse,
-                                            aSecondVeryLongParam: AVeryLongTypeThatYouNeedToUse,
-                                            aThirdVeryLongParam: AVeryLongTypeThatYouNeedToUse)
-                                            : AReturnType =
-        // ... the body of the method
-    member _.LongMethodWithLotsOfCurrifiedParams(aVeryLongParam: AVeryLongTypeThatYouNeedToUse)
-                                                (aSecondVeryLongParam: AVeryLongTypeThatYouNeedToUse)
-                                                (aThirdVeryLongParam: AVeryLongTypeThatYouNeedToUse)
-                                                =
-        // ... the body of the method
+    member _.LongMethodWithLotsOfParameters
+        (
+            aVeryLongParam: AVeryLongTypeThatYouNeedToUse,
+            aSecondVeryLongParam: AVeryLongTypeThatYouNeedToUse,
+            aThirdVeryLongParam: AVeryLongTypeThatYouNeedToUse
+        ) : AReturnType =
+            // ... the body of the method
```

Comment:
Same reasoning as previous comment

```suggestion
    member _.LongMethodWithLotsOfParameters
        (aVeryLongParam: AVeryLongTypeThatYouNeedToUse,
         aSecondVeryLongParam: AVeryLongTypeThatYouNeedToUse,
         aThirdVeryLongParam: AVeryLongTypeThatYouNeedToUse)
        : AReturnType =
            // ... the body of the method
```

Reply:
@knocte Here's what I mean (/cc @cartermp). If you have a sizable return type, even with your original suggestion, the body is not separated well from the parameter list.

Reply:
Sure, but this circumstance feels incredibly rare though.

Reply:
Let's keep the original suggestion then to be consistent with [this](https://github.com/dotnet/docs/pull/21690/files#r529190027)